datacommons/utils.py

```diff
@@ -141,3 +108,55 @@ def format_response(response, compress=False):
   if compress:
     payload = zlib.decompress(base64.b64decode(payload), 16 + zlib.MAX_WBITS)
   return json.loads(payload)
+
+def format_expand_payload(payload, new_key, must_exist=[]):
+  """ Formats expand type payloads into dicts from dcids to lists of values. """
+  # Create the results dictionary from payload
+  results = defaultdict(list)
+  for entry in payload:
+    if 'dcid' in entry and new_key in entry:
+      dcid = entry['dcid']
+      results[dcid].append(entry[new_key])
+
+  # Ensure all dcids in must_exist have some entry in results.
+  for dcid in must_exist:
+    results[dcid]
+  return dict(results)
+
+def convert_dcids_type(dcids):
+  """ Amends dcids list type and creates the approprate request dcids list. """
+  # Format the dcids list.
+  if isinstance(dcids, str):
```

Comment:
Might not want the dcids here to be string or only list or pandas series. So this function could be just:

list(dcids)

Reply:
Good point, removed